### DISCUSSION TITLE
Cookie blocked due to "not matching" domains

### DISCUSSION THREAD

**Discussion by hjmace** [GoogleChromeLabs/ps-analysis-tool#586] [Support Forum] [answered]
While learning how to use the PSAT for auditing I came upon this error: "This cookie was blocked because the domain in the web address for this request was neither an exact match with the cookie's domain attribute nor a subdomain of it."

This brings the question of why it was blocked, because it's the right domain.
![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/165011827/ca518854-16db-4baa-a6b0-f53599bb68f4)

**Comment by milindmore22** [comment, marked as answer]
Hello @hjmace 

Based on the analysis of your screenshot, it seems the cookie has a Lax security setting.

The cookie is most likely from appmaster.io, but it might be getting set on a different website you visited.
Lax cookies offer some security by only being sent when you directly visit the website that set them, like following a link to appmaster.io. This is the default behavior if the `SameSite` attribute is not specified.

To pinpoint the exact request that might be adding the blocked cookie, We will recommend checking our guide for filtering [network request for given cookie](https://github.com/GoogleChromeLabs/ps-analysis-tool/wiki/Cookies-Table#network-requests-with-a-given-cookie)
